src/app/body/all-products/filters/rating-filter/rating-filter.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+// @ts-ignore
```

Comment:
// @ts-ignore
It's not a good idea to avoid ts lint errors.

Reply:
fixed